Assets/MixedRealityToolkit.SDK/Experimental/HandCoach/Scripts/MoveToTarget.cs

```diff
@@ -0,0 +1,197 @@
+{
+    /// <summary>
+    /// This class provides functionality to move the hand hint from a tracking position to a target position over time.
+    /// </summary>
+    public class MoveToTarget : MonoBehaviour
+    {
+        [Tooltip("Object to track.")]
+        [SerializeField]
+        private GameObject trackingObject = null;
+
+        /// <summary>
+        /// Object to track.
+        /// </summary>
+        public GameObject TrackingObject
+        {
+            get
+            {
+                return trackingObject;
+            }
+            protected set
+            {
+                trackingObject = value;
+            }
+        }
+
+        [Tooltip("Target to move to.")]
+        [SerializeField]
+        private GameObject targetObject = null;
+
+        /// <summary>
+        /// Target to move to.
+        /// </summary>
+        public GameObject TargetObject
+        {
+            get
+            {
+                return targetObject;
+            }
+            protected set
+            {
+                targetObject = value;
+            }
+        }
+
+
+        [Tooltip("Shared parent between tracking and target objects used for relative local positions.")]
+        [SerializeField]
+        private GameObject rootObject = null;
+
+        /// <summary>
+        /// Shared parent between tracking and target objects used for relative local positions.
+        /// </summary>
+        public GameObject RootObject
+        {
+            get
+            {
+                return rootObject;
+            }
+            protected set
+            {
+                rootObject = value;
+            }
+        }
+
+        [Tooltip("Duration of move from tracking object to target object in seconds.")]
+        [SerializeField]
+        private float duration = 1.38f;
+
+        /// <summary>
+        /// Duration of move from tracking object to target object in seconds.
+        /// </summary>
+        public float Duration
+        {
+            get
+            {
+                return duration;
+            }
+            protected set
+            {
+                duration = value;
+            }
+        }
+
+        [Tooltip("Tunable offset to get the GameObject to arrive at the right target position.")]
+        [SerializeField]
+        private Vector3 targetOffset = new Vector3(0f, 0f, 0f);
+
+        /// <summary>
+        /// Tunable offset to get the GameObject to arrive at the right target position.
+        /// </summary>
+        public Vector3 TargetOffset
+        {
+            get
+            {
+                return targetOffset;
+            }
+            protected set
+            {
+                targetOffset = value;
+            }
+        }
+
+        [Tooltip("Lerp curve.")]
+        [SerializeField]
+        private AnimationCurve animationCurve = AnimationCurve.Linear(0f, 0f, 1f, 1f);
+
+        /// <summary>
+        /// Lerp curve.
```

Comment:
Could you describe what the animation controls? Lerp curve to me is similar to "animation curve" in that you're describing what this variable is, but not how it is used, or how somebody else might use it.